### PR TITLE
BETA: Register temelkov.is-a.dev

### PR DESCRIPTION
**Pull request by Temelkov**
Added `temelkov.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/temelkov [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Stanislav Temelkov Professional Portfolio",
+  "repo": "https://github.com/Temelkov/TEMELKOV.IS-A.DEV",
+  "owner": {
+    "username": "temelkov",
+    "email": "stanislav.temelkov@hotmail.com"
+  },
+  "record": {
+    "CNAME": "temelkov-tk-2022.vercel.app"
```
Comment (Stef-00012) [inline domains/temelkov:9]: Vercel is not supported yet (might be soon)

File: domains/temelkov.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Temelkov",
+        "email": "stanislav.temelkov@hotmail.com"
+    },
+    "record": {
+        "CNAME": "temelkov-tk-2022.vercel.app"
```
Comment (Stef-00012) [inline domains/temelkov.json:7]: Vercel is not supported yet (might be soon)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/temelkov (hunk 1 above)
Comment (Temelkov) [reply domains/temelkov:9]: Okey, do this with this CNAME: https://temelkov.github.io/TEMELKOV-IS-A-DEV/

Comment (Temelkov) [reply domains/temelkov:9]: @Stef-00012

File: domains/temelkov.json (hunk 2 above)
Comment (Temelkov) [reply domains/temelkov.json]: @Stef-00012 I Change it. Please merge!

File: domains/temelkov
Comment (Stef-00012) [inline domains/temelkov:1]: file name should end with .json

File: domains/temelkov [hunk 3]
```diff
@@ -6,6 +6,6 @@
     "email": "stanislav.temelkov@hotmail.com"
   },
   "record": {
-    "CNAME": "temelkov-tk-2022.vercel.app"
+    "CNAME": "temelkov.github.io/TEMELKOV-IS-A-DEV"
```
Comment (Stef-00012) [inline domains/temelkov:9]: CNAME can not contain /

File: domains/temelkov.json [hunk 4]
```diff
@@ -4,6 +4,6 @@
         "email": "stanislav.temelkov@hotmail.com"
     },
     "record": {
-        "CNAME": "temelkov-tk-2022.vercel.app"
+        "CNAME": "temelkov.github.io/TEMELKOV-IS-A-DEV"
```
Comment (Stef-00012) [inline domains/temelkov.json:7]: CNAME can not contain /

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/temelkov.json (hunk 4 above)
Comment (Temelkov) [reply domains/temelkov.json]: @Stef-00012 DONE!

**Comment by Temelkov** [comment]
@Stef-00012 PING!

**Comment by orxngc** [comment]
Your site is inaccessible.

**Comment by Temelkov** [comment]
> Your site is inaccessible.

https://temelkov.github.io/TEMELKOV-IS-A-DEV/

**Pull request closed without merging**